app/views/organizations/organization_profiles/_form.html.erb

```diff
@@ -50,7 +36,6 @@
             <div class="alert alert-danger mt-1" role="alert"><%= error.message %></div>
           <% end %>
 
```

Comment:
Can we keep this label? Province/State is clearer than Province state in my opinion.

Reply:
Hey @kasugaijin! I added the naming of the fields on the locale file, so now bootstrap_form should get the desired label. 

Its on the last commit the changes, could you please let me what you know? If you think the label should go back anyways, lemme know and Ill revert it.

Im gonna upload another image using the new translation

Reply:
@yuricarvalhop ah yes I see how that works now! The screenshots through me off. Yes it looks good.